src/ddmd/statement.d

```diff
@@ -47,16 +47,22 @@ import ddmd.staticassert;
 import ddmd.tokens;
 import ddmd.visitor;
 
+/*****************************************
+ * CTFE requires FuncDeclaration::labtab for the interpretation.
+ * So fixing the label name inside in/out contracts is necessary
+ * for the uniqueness in labtab.
```

Comment:
* To me, this description sounds more like the reason why `fixupLabelName` is needed, not what it does
* Try to have a short summary [1]

[1] http://dlang.org/spec/ddoc.html#summary

Reply:
@jacob-carlborg could you please follow up on this with another PR? Thx!

Reply:
When did it become the norm that reviewers should fix issues in the PR? I will assume that when I create a PR next time 😉.

Reply:
I was not referring to Andrei fixing things in this PR. I was referring to his comment about me creating an other PR fixing what I commented on.

Reply:
@jacob-carlborg you got a point, thanks for your response. @WalterBright's pull requests are "special" because as the creator and most prolific contributor to all aspects of the language, the allocation of his time is very important to D. So to the extent a pull request of his moves the ball forward and does not create technical debt, I'll allow them even if they could be made better. Though generally it's reasonable to ask him to take his pull requests to perfection, let us all improve on his work. Thanks!

Reply:
That's true, Walter's time is really valuable. At the same time I think you and Walter should lead by example. 🤔 it's not easy to balance these things.